### DISCUSSION TITLE
github contribution graph not showing private commits?

### DISCUSSION THREAD

**Discussion by samuelmehr** [community/community#46060] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

I just went to update my profile page and noticed that most of my GitHub contributions are not showing up. We have 20-30 private repositories in my organisation account and I contribute to them regularly. Contributions to private repos used to always display, and "show contributions to private repositories" is enabled on my account.

Just now I made a test commit to a public repo in the same org account just to make sure it's working, and it did appear instantly as a little green contribution in my profile, but private ones do not.

Did something change about the ability to display these? Perhaps I need to enable something in the private repo settings that didn't used to be required?

**Comment by radar-blip** [comment, marked as answer]
It's possible you might find an answer [here](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/managing-contribution-settings-on-your-profile/why-are-my-contributions-not-showing-up-on-my-profile).

**Reply by samuelmehr** [reply]
Thanks, but none of the scenarios mentioned there apply here, since commits to public repos display normally from the same machine / git configuration where private commits don't appear from. So unless I'm missing something I think there's a bug here.

**Reply by radar-blip** [reply]
Have you checked this in your settings?

![pricont](https://user-images.githubusercontent.com/39196767/216284422-4e6db31d-c922-44ca-b1db-80d6a3a01347.png)

**Reply by samuelmehr** [reply]
Yes, of course, it's enabled.

**Reply by radar-blip** [reply]
Thought so. I cant think of anything else that would cause that to happen.
Could be a bug like you said.
Here's a link to file a bug report if you'd like to: [Bug-Report](https://support.github.com/contact/bug-report)

**Reply by samuelmehr** [reply]
Sure, I'll submit one, maybe this is not yet a known issue? It's pretty weird because these all used to appear normally. Doesn't really matter in terms of work but I like the little green mosaic 😆

**Reply by samuelmehr** [reply]
Turns out your initial answer did solve it, apparently I have a lot of older commits under an old email address which I'd removed from my account. My mistake, sorry.

**Reply by radar-blip** [reply]
No problem. Glad you got it worked out!

**Reply by noorhashem** [reply]
Did you file a bug report and did it get solved? I am currently having the exact same issue.

**Reply by noorhashem** [reply]
So I contacted the Github support and they manually rebuilt my graph, so looks like it was just a bit sluggish, now they show up.

**Comment by chetandabli** [comment]
steps to enable showing private contributions is

1.) go to the profile
2.) go to your contributions chart
3.) click on Contribution settings 
4.) enable Private contributions by clicking on it.

that's it. it will show all the contributions.

thanks

**Reply by samuelmehr** [reply]
This is enabled, and they don't appear.

**Reply by samuelmehr** [reply]
![image](https://user-images.githubusercontent.com/41483946/216283095-fba4f99f-098b-4b83-bbc8-dc56e39ea65f.png)
(I made several hundred commits in 2022, almost entirely to private repos)

**Reply by chetandabli** [reply]
Commits will not show. only merged contributions are shown on the contribution graph. but not sure please check this thing also.

**Reply by samuelmehr** [reply]
Commits to main on public repos do show; commits to main on private repos used to show but now do not.

**Reply by Comi15** [reply]
Something must be bugged, because I just tested it and it works fine when I commit to my private repos on main it shows it in the contribution graph if the private contributions option is enabled.
![Screenshot (26)](https://user-images.githubusercontent.com/101349980/216341869-faecfc44-59a9-49c7-97cf-fc4bf3a81a44.png)

**Reply by exilonium** [reply]
the commits on the private repo shows only if they are on the default branch
